chef/cookbooks/nova/templates/default/libvirtd.conf.erb

```diff
@@ -271,13 +270,13 @@ auth_tcp = "<%= @libvirtd_auth_tcp %>"
 #
 # XXX this isn't actually enforced yet, only the per-client
 # limit is used so far
-#max_requests = 20
+max_requests = <%= @libvirtd_max_clients %>
 
 # Limit on concurrent requests from a single client
 # connection. To avoid one client monopolizing the server
 # this should be a small fraction of the global max_requests
 # and max_workers parameter
-#max_client_requests = 5
+max_client_requests = <%= @libvirtd_prio_workers %>
```

Comment:
This implies the number of priority workers is related to max client requests and min workers, but it's not.  Do you think it is required to change priority workers?

I think these changes depend on how nova manages connections to libvirtd.  If only one connection when nova initializes, then bumping max_workers and max_client_requests should be all thats needed.  If there is one connection per operartion, then bump max_clients and max_workers.

Note that max_clients has changed to 5000 upstream, and addtional knobs have been added.  These changes are in SLES12:
# The maximum number of concurrent client connections to allow
# over all sockets combined.
#max_clients = 5000

# The maximum length of queue of connections waiting to be
# accepted by the daemon. Note, that some protocols supporting
# retransmission may obey this so that a later reattempt at
# connection succeeds.
#max_queued_clients = 1000

# The maximum length of queue of accepted but not yet not
# authenticated clients. The default value is zero, meaning
# the feature is disabled.
#max_anonymous_clients = 20